start-server-https.cmd

```diff
@@ -0,0 +1,17 @@
+set myVar="C:\Users\ValMe\Documents\GitHub\hpc-datastore"
```

Comment:
enviromental dependent information should not be in git.
Similar thinks is solved that you create template and commit it. Whilst file that should is derived from a template is mentioned in .gitignore.